index.d.ts

```diff
@@ -84,3 +84,41 @@ const ab: Merge<Foo, Bar> = {a: 1, b: 2};
 ```
 */
 export type Merge<FirstType, SecondType> = Omit<FirstType, Extract<keyof FirstType, keyof SecondType>> & SecondType;
+
+/**
+Create a new type from an object type extracting just props
+
```

Comment:
Should we maybe use `@see` to link between these methods since they are related? http://usejsdoc.org/tags-see.html

Reply:
done :+1: